apps/vc-api/src/vc-api/credentials/dtos/verify-credential.dto.ts

```diff
@@ -25,21 +23,4 @@ export class VerifyCredentialDto {
       'https://w3c-ccg.github.io/vc-api/issuer.html#operation/issueCredential'
   })
   verifiableCredential: VerifiableCredentialDto;
-
-  /**
-   * Parameters for verifying a verifiable credential or a verifiable presentation
-   * https://w3c-ccg.github.io/vc-api/verifier.html#operation/verifyCredential
-   * https://w3c-ccg.github.io/vc-api/verifier.html#operation/verifyPresentation
-   */
-  @IsObject()
-  @IsDefined()
-  @ValidateNested()
-  @Type(() => VerifyOptionsDto)
-  @ApiProperty({
-    description:
-      'Parameters for verifying a verifiable credential or a verifiable presentation ' +
-      'https://w3c-ccg.github.io/vc-api/verifier.html#operation/verifyCredential ' +
-      'https://w3c-ccg.github.io/vc-api/verifier.html#operation/verifyPresentation'
-  })
-  options: VerifyOptionsDto;
```

Comment:
What was the thinking in removing the `options` parameter here? "verify credential" does have an options object, though it is different from the current `VerifyOptionsDto` that we're using (the only option for verifying a credential is `returnCredential` currently). https://w3c-ccg.github.io/vc-api/#verify-credential

Reply:
@jrhender We need to update the verify options DTO to accept only 1 param `returnCredential` and also update the  VerificationResultDTO to return the credential of the `returnCredential` is true.
We need to have separate verify-options.dto for `credential verify` and `presentation verify` methods because their `options` parameter requirements are different as we see in the spec.

Reply:
@ShikharBhatt yes, I agree those are the changes we should make. Was the thinking that we should remove the `options` parameter until we make those changes?
I think this set of changes is separate from the work of integrating Credo so should probably be done in a separate PR. Do you agree? If so, I can make a separate GitHub issue to track.

Reply:
@jrhender Maybe we should then keep the `options` parameter for now, even-though it's not going to be actually consumed anywhere in the code (in its current structure). After understanding the spec, I now realize that `options` parameter is needed and shouldn't have been removed in the first place.

And yes, it would make sense to track this particular change as a separate issue.

Let me know if it makes sense.
Thanks.

Reply:
@ShikharBhatt good points. I think you were right then 😄 . We should remove the `options` parameter from the `VerifyCredentialDto`

Reply:
@jrhender Gotcha!!!
Will push that final change then.
Thanks!!!

Reply:
Made an issue to track this: https://github.com/openwallet-foundation-labs/vc-api/issues/35